Formula/socket_vmnet.rb

```diff
@@ -25,18 +25,16 @@ def install
     system "make", "install.bin", "install.doc", "VERSION=#{version}", "PREFIX=#{prefix}"
   end
 
-  def caveats
-    <<~EOS
-      To install an optional launchd service, run the following command (sudo is necessary):
-      sudo brew services start socket_vmnet
-    EOS
```

Comment:
Why remove this?
sudo is necessary for `socket_vmnet`.

Reply:
https://github.com/Homebrew/homebrew-core/pull/125725#discussion_r1137041739

Reply:
https://github.com/Homebrew/homebrew-core/pull/125725#discussion_r1137917064